### ISSUE TITLE
automation

### ISSUE THREAD

**Issue by adamjakab** [adamjakab/BeetsPluginTemplate#1]
add shell script to automate the steps described in the README doc

**Comment by adamjakab** [comment]
i mean a python script

**Comment by GuilhermeHideki** [comment]
What do you think about moving to cookiecutter? https://cookiecutter.readthedocs.io/en/latest/README.html

**Comment by adamjakab** [comment]
> What do you think about moving to cookiecutter? 

Yes! I think that if I ever get around to do it then Cookiecutter is in pole position! Thanks for the tip. It will surely give a more structured way to do it than my home-baked script.
